### PR TITLE
More details create credit and create tax items endpoints

### PR DESCRIPTION
**Pull request by KBbitsP**
Added details about create credit and create tax items endpoints not saving linked invoice item and subscription id.

### CODE REVIEW

**Comment by KBbitsP** [comment]
@reshmabidikar

File: source/includes/_credit.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Basic endpoints to create and retrieve credit invoice items
 
 ### Create credits
 
-Create one or more credits for a specified account. These credits will appear as `invoiceItem`s. They may be added to an existing DRAFT invoice, or they may result in the creation of a new invoice. 
+Create one or more credits for a specified account. These credits will appear as `invoiceItem`s. They may be added to an existing DRAFT invoice, or they may result in the creation of a new invoice. Also, it is worth noting, that the credits are added either at account level OR invoice level (if invoiceID is specified) and not linked to any existing `invoiceItem`s. Consequently, if you pass certain fields like `linkedInvoiceItemId`, `subscriptionID` while creating the credit, they will not reflect in the database and the same will not be visible in subsequent Get requests.
```
Comment (reshmabidikar) [inline source/includes/_credit.md:18]:
```suggestion
Create one or more credits for a specified account. These credits will appear as `invoiceItem`s. They may be added to an existing DRAFT invoice, or they may result in the creation of a new invoice. While adding a credit to an existing invoice, specifying fields like `linkedInvoiceItemId`, `subscriptionId` has no effect. These fields do not get saved in the database and will not be returned in subsequent `GET` requests
```

File: source/includes/_invoice.md [hunk 2]
```diff
@@ -593,7 +593,7 @@ If successful, returns a status code of 201 and an invoice item object.
 
 ### Create tax items
 
-Normally, tax items are added to an invoice by a plugin, which intercepts the invoice during its creation and adds the required tax items on the fly. However, sometimes you may want to add tax items directly. This API adds a tax item to an account, which will result in the creation of a new invoice.
+Normally, tax items are added to an invoice by a plugin, which intercepts the invoice during its creation and adds the required tax items on the fly. However, sometimes you may want to add tax items directly. This API adds a tax item to an account, which will result in the creation of a new invoice. Also, it is worth noting, that the tax items are not linked to any existing `invoiceItem`s. Consequently, if you pass certain fields like `linkedInvoiceItemId`, `subscriptionID` while creating the tax item, they will not reflect in the database and the same will not be visible in subsequent Get requests.
```
Comment (reshmabidikar) [inline source/includes/_invoice.md:596]:
```suggestion
Normally, tax items are added to an invoice by a plugin, which intercepts the invoice during its creation and adds the required tax items on the fly. However, sometimes you may want to add tax items directly. This API adds a tax item to an account, which will result in the creation of a new invoice. Also, it is worth noting, that the tax items are not linked to any existing `invoiceItem`s. Consequently, specifying fields like `linkedInvoiceItemId`, `subscriptionId` while creating the tax item has no effect. These fields do not get saved on the database and will not be returned in subsequent `GET` requests.
```

**Review by pierre — COMMENTED** [review]
Merging for now, but see https://github.com/killbill/killbill-docs/pull/379/files#r902209998

If indeed the behavior is different between invoice plugin and tax API, we should probably fix this (not urgent).

**Pull request merged**